tools/mod/go.mod

```diff
@@ -1,6 +1,8 @@
 module go.etcd.io/etcd/tools/v3
 
-go 1.20
+go 1.21
```

Comment:
Please ensure the project stays on go 1.20.  We are not ready to update to go 1.21 and that will be tracked separately when we are.

Reply:
This was a mistake. Will be fixed soon.

Reply:
Please raise a separate PR to bump golang to 1.20.8 firstly. thx @jmhbnz @sharathsivakumar

Reply:
It's blocking all other PRs.  Let me bump golang version myself right now.

Reply:
FYI. https://github.com/etcd-io/etcd/pull/16554

Reply:
https://github.com/etcd-io/etcd/pull/16554 has already been merged, please rebase this PR, thx